generate-docs/json/excel_1_11/excel.api.json

```diff
@@ -225456,6 +225467,21 @@
                 }
               ]
             },
+            {
+              "kind": "Class",
+              "canonicalReference": "excel!Excel.RangeCustom:class",
```

Comment:
I think the RangeCustom class slipped into DefinitelyTyped. Let's remove that.

Reply:
Thanks for catching this! Working on the fix now.